andela/the_spinners-frontend
# #171692118 Integrate view of all notifications

#### What does this PR do?
- make all notifications available including ones that are read
#### Description of Task to be completed?
#### How should this be manually tested?
`checkout this branch then try if you have some notifications you should see them. 
Unread notifications should be in a different color` 
#### Any background context you want to provide?
- Feedback received in the last sprint 
#### What are the relevant pivotal tracker stories?
[#171692118](https://www.pivotaltracker.com/story/show/171692118)
#### Screenshots (if appropriate)
<img width="369" alt="Screen Shot 2020-03-09 at 12 13 17" src="https://user-images.githubusercontent.com/52409462/76203499-68bd1b80-61ff-11ea-9a35-d8abc3c6e4ef.png">

#### Questions:
